src/Strings/en-us/Resources.resw

```diff
@@ -349,4 +349,32 @@
   <data name="WidgetScreenshotAltDisplaySSKeychain" xml:space="preserve">
     <value>SSH Keychain widget preview image</value>
   </data>
+  <data name="DevHomeAzureDisplayName" xml:space="preserve">

```

Comment:
Can we get localized info from the store? I worry about overriding someone's localized names.

Reply:
So I have these for the case where the user doesn't have internet access. We'd fallback to using these. In a future PR I'd be using the store api url to get the info

Reply:
Removed and now using WinGet to get display name, description and publisher name